shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -5,6 +5,15 @@
 
 public class ConfigurationService {
 
+    private static ConfigurationService configurationService;
+
+    public static ConfigurationService getInstance() {
```

Comment:
Is this used anywhere?

Reply:
Good spot ... not used any more, changed my mind on how to inject the config service.

Reply:
Yes ... in `StateMachine.java`, line 106 ... it's a large diff, so may not show by default.

Reply:
Ah yes that will be it